casanovo/casanovo.py

```
@@ -374,6 +391,129 @@ def train(
+
+            total_iters = warmup + n_iter
+            for i in range(total_iters):
+                results_path = (
+                    output_path
+                    / f"{output_root_name}_bench_{batch_size}_{n_beams}_{i}.mztab"
+                )
+                if torch.cuda.is_available():
+                    torch.cuda.reset_peak_memory_stats()
+
+                with ModelRunner(
+                    cfg, model_path, output_path, None, False
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)